python-avd/pyavd/_eos_designs/shared_utils/filtered_tenants.py

```diff
@@ -203,6 +203,9 @@ def filtered_vrfs(
             vrf.l3_interfaces = self.filtered_l3_interfaces(vrf)
             vrf.l3_port_channels = self.filtered_l3_port_channels(vrf)
             vrf.loopbacks = vrf.loopbacks._filtered(lambda loopback: loopback.node == self.hostname)
+            vrf.aggregate_addresses = vrf.aggregate_addresses._filtered(
+                lambda aggregate_address: self.match_regexes(aggregate_address.nodes, self.hostname)
```

Comment:
Please create a helper method `self.match_nodes` we can use everywhere. This will be called once per VRF, and a lambda will have a small performance cost every time. (I know we have it above, but that can be improved seperately)